packages/metrics/src/features/vsExport/handler.ts

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
@sadmananik this file needs some test coverage please

Reply:
Added unit tests for getAllVSExport route handler